recipe_scrapers/wikicookbook.py

```diff
@@ -23,21 +27,23 @@ def yields(self):
         )
 
     def image(self):
-        image = self.soup.find("a", {"class": "image"}).find("img", {"src": True})
+        image = self.soup.find("a", {"class": "mw-file-description"}).find(
+            "img", {"src": True}
+        )
         return image["src"] if image else None
 
     def ingredients(self):
-        ingredients = (
-            self.soup.find("span", {"id": "Ingredients"}).find_next("ul").findAll("li")
-        )
-
+        ingredients_section = self.soup.find("h2", {"id": "Ingredients"})
+        if not ingredients_section:
+            raise ElementNotFoundInHtml(element="//h2[text()='Ingredients']")
```

Comment:
My mistake here; this should be an `@id` (ID attribute) selector instead of a `text()` content selector:

```suggestion
            raise ElementNotFoundInHtml(element="//h2[@id='Ingredients']")
```